public/js/components/Editor.css

```diff
@@ -27,23 +27,28 @@
   width: 60px;
   height: 12px;
   position: absolute;
-  top: 0;
+  top: 1px;
   right: -4px;
 }
 
 .editor.new-breakpoint.breakpoint-disabled svg {
   opacity: 0.3;
 }
 
+.search-bar ~ .editor-mount {
+  height: calc(100% - 58px)
+}
+
 .editor-mount,
 .CodeMirror {
   width: 100%;
-  height: 100%;
+  height: calc(100% - 17px);
```

Comment:
I feel like we have too many little hacks around the editor area (1px offset here and there). I would *love* it if you could take a few minutes to think about about the editor area and possibly refactor it a little bit to avoid this. Maybe a few more uses of flexbox would help. I feel like it's getting way too brittle.

I'm fine including that in this PR, it's related to the UI changes you want to make. It's hard for me to approve yet another hardcoded value :)